src/main/java/com/enterprise/eventmanagementsystem/controller/EventController.java

```diff
@@ -52,7 +52,7 @@ public String viewHomePage(Model model) throws Exception {
     public String showNewEventForm(Model model) throws Exception {
         Event event = new Event();
         model.addAttribute("event", event);
-        model.addAttribute("eventsSearchableData", generateEventsSearchableData());
+        model.addAttribute("eventsSearchableData", eventService.generateEventsSearchableData());
```

Comment:
Once again, I'm just calling the service to get the searchable data now, instead of the controller method.